assets/js/googlesitekit/datastore/forms/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * core/forms Data store: values
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Data from 'googlesitekit-data';
+import invariant from 'invariant';
+import { STORE_NAME } from './constants';
+
+// Actions
+const SET_FORM_VALUES = 'SET_FORM_VALUES';
+
+export { STORE_NAME };
+
+export const INITIAL_STATE = {
+};
+
+export const actions = {
+	setValues( formName, formData ) {
+		invariant( formName, 'form name is required.' );
```

Comment:
The field referenced in the message should match the name of the parameter. Also it's a good idea to make the message a bit more specific to where it's used. That way, when we see the error we know where it's happening.  Otherwise `x is required` is not very useful unless you're unit testing that exact function. Take a look at some of the other `invariant` uses in the codebase to see what I mean. What you have here is consistent with much of the invariant use we have, so this is a suggestion I have for everyone.

In this case though, something like `formName is required for setting values.` would be great.

Also, `formData` should be covered by `invariant` as well as a required valid value 😄